plugins/chat/app/services/chat/search_chatable.rb

```diff
@@ -91,10 +91,12 @@ def search_users(params, guardian)
     end
 
     def search_groups(params, guardian)
+      # NOTE: Do NOT eager load users here (e.g. `.includes(users: ...)`).
+      # Groups can have 100k+ members and loading them causes request timeouts.
+      # The serializer uses SQL COUNT queries instead.
```

Comment:
If the serializer is doing a count, it would result in an N+1 queries problem though it is limited to `SEARCH_RESULT_LIMIT` queries in this case.